tests/openthread/Makefile

```diff
@@ -11,43 +12,47 @@ RIOTBASE ?= $(CURDIR)/../..
 # Comment this out to disable code in RIOT that does safety checking
 # which is not needed in a production environment but helps in the
 # development process:
-CFLAGS += -DDEVELHELP -Wall
+#CFLAGS += -DDEVELHELP -Wall
 
 # Change this to 0 show compiler invocation lines by default:
 QUIET ?= 1
 
 USEPKG += openthread
-USEMODULE += openthread_contrib
-USEMODULE += libmbedcrypto
-USEMODULE += libopenthread
-USEMODULE += libopenthread-cli
-USEMODULE += xtimer
+OPENTHREAD_TYPE ?= ftd
+ifeq ($(OPENTHREAD_TYPE),mtd)
+    # MTD: A Minimal Thread Device does not have router functionality
+    # compiled in. As a result, it is not necessary to configure the
+    # routerrole on an MTD. At the same time, an MTD may or may not be sleepy.
+    USEMODULE += openthread-mtd
+    USEMODULE += openthread-cli-mtd
+else
+    # ftd: A Full Thread Device has router functionality compiled in
+    USEMODULE += openthread-ftd
+    USEMODULE += openthread-cli-ftd
+endif
+
+#Define PANID and CHANNEL used by default
+#CFLAGS += -DOPENTHREAD_PANID=0xbeef -DOPENTHREAD_CHANNEL=11
```

Comment:
I think it would be even simpler to use special variables for each with default values:
```make
OPENTHREAD_PANID ?= 0xbeef
OPENTHREAD_CHANNEL ?= 11

CFLAGS += -DOPENTHREAD_PANID=${OPENTHREAD_PANID} -DOPENTHREAD_CHANNEL=${OPENTHREAD_CHANNEL}
```
Then user could pass the variable independently when building the firmware and without having to edit the Makefile
```
make BOARD=samr21-xpro OPENTHREAD_PANID=0xaaaa OPENTHREAD_TYPE=ftd flash term
```